docs/Project-Configuration/Project-Commands.md

```diff
@@ -397,18 +397,38 @@ Any updates to the expansions made with this command will only persist for the d
   params:
     ignore_missing_file: true
     file: src/ec2_artifacts.yml
+    redact_file_expansions: true
 
 - command: expansions.update
   params:
     updates:
     - key: artifact_url
       value: http://s3/static-artifacts.tgz
+
+- command: expansions.update
+  params:
+    updates:
+    - key: artifact_url
+      concat: tgz
+
+- command: expansions.update
+  params:
+    updates:
+    - key: artifact_url
+      value: http://s3/static-artifacts.tgz
+      redact: true
 ```
 
 Parameters:
 
--   `updates`: key-value pairs for updating the task's parameters
+-   `updates`: a list of expansions to update.
+        - `key`: the expansion key to update. (required)
+        - `value`: the new value for the expansion.
+        - `concat`: the string to concatenate to the existing value. Per
+           update, only `value or `concat` can be set.
+        - `redact`: if true, the expansion will be redacted in the task logs.
```

Comment:
A very small edge case, but if we update a function var to be redacted with expansions.update, I would think that at the end of the function, it would revert back to its original value + original redacted setting. Would it be worth calling that out somewhere?

Reply:
I think with the new wording, it should be clear that the redaction is at an update level rather than persisting the keys type implementation

Reply:
Thanks, the new wording is clearer. To make sure there's no ambiguity, can we clarify that you can only use `redact: true` to enable redaction, and that `redact: false` won't un-redact? For example, if you have a private var `my_var` as an expansion, then you can't un-redact it using:
```
my_var: <same_value>
redact: false # this setting doesn't do anything, my_var is still redacted
```